src/lib/components/ng-http-loader.component.ts

```diff
@@ -20,7 +20,7 @@ import { Spinkit } from '../spinkits';
     styleUrls: ['./ng-http-loader.component.scss']
 })
 export class NgHttpLoaderComponent implements OnDestroy, OnInit {
-    public isSpinnerVisible: boolean;
+    public isSpinnerVisible = of(false);
```

Comment:
I'm a bit confused why this isn't an `new BehaviourSubject<boolean>(false)` that you `.next(true|false)` rather than mutating the reference. I'm quite new to rxjs, but re-assigning an observable rather than using it as a state wrapper feels like a code smell.

Reply:
You're right, that will be much better !